server/src/resolvers/Mutation.js

```diff
@@ -2,10 +2,16 @@ const bcrypt = require('bcryptjs')
 const jwt = require('jsonwebtoken')
 const { APP_SECRET, getUserId } = require('../utils')
 
-function post(parent, args, context) {
+function post(parent, { url, description }, context) {
+  const userId = getUserId(context)
```

Comment:
I think we'll have to be a little clever and make a separate branch where this change hasn't been made for the tutorial intro 🤔

Reply:
We might just opt to not throw in our final solution?

Reply:
I think it's safer if we update the starter branch with the basic stuff from after the getting started step. Happy to do that once we merge this PR :)

Reply:
well for starter we only need to remove this line. That will be used for codeload right?